WFP-VAM/prism-frontend
# Fix url replace error COUNTRY=cambodia

fixes #552 

**How to test:** Using cambodia
- Select a WMS Layer.**rainfall_dekad**
- Select a PointDataLayer, like Kobo or EWS.
- Verify that the Url is correctly generated
